quickwit-cli/src/cli.yaml

```diff
@@ -90,6 +90,29 @@ subcommands:
                 help: Filters out documents after that timestamp (time-series indexes only)
                 long: end-timestamp
                 value_name: TIMESTAMP
+    - serve:
+        about: Serve a REST API to search an index
+        args:
+            - metastore-uri:
```

Comment:
Is this change only for `serve` or have we agreed to change the external view of users from `index-uri` to `metastore-uri` ?

Reply:
That's a good question... One benefit of the metastore like this is that we allow for more than one index.

I guess we could also offer more than one index-uri?
Let me try it.

Reply:
As for me, this is the way to go because we want to serve several indexes. 
But the user starts by giving an index-uri... and we don't explain to him that we deduce from it a metastore uri. And maybe it would be simpler to say to the user: "give a us a metastore uri and a index name"....